### DISCUSSION TITLE
Neper2CAE

### DISCUSSION THREAD

**Discussion by kmmm70** [neperfepx/neper#453] [Q&A] [answered]
Hi All,

Does anyone have a chance to use it?
I've got an error while trying to run the python script created by Neper. Please see the attached file. 

Thanks.
![error1](https://user-images.githubusercontent.com/57537961/165272039-5ef0e630-4dd9-4e2b-a1a8-cd441da9be39.png)

**Comment by rquey** [comment, marked as answer]
https://github.com/ngrilli/Neper2CAE may be a better place to ask, as we do not use nor provide support for external modules.
